collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloaderUseCases.kt

```diff
@@ -0,0 +1,22 @@
+package org.odk.collect.android.formmanagement
+
+import org.odk.collect.android.utilities.FileUtils
+import org.odk.collect.android.utilities.FileUtils.LAST_SAVED_FILENAME
+import org.odk.collect.forms.FormsRepository
+import java.io.File
+
+object ServerFormDownloaderUseCases {
```

Comment:
Love it! Maybe we should move `FormMediaDownloader.download` in here as well?

Reply:
Yeah, we can do that, done.